manifests/b/BraveSoftware/BraveBrowser/102.0.5005.61/BraveSoftware.BraveBrowser.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.BraveBrowser
+PackageVersion: 102.0.5005.61
```

```suggestion
PackageVersion: 102.1.39.111
```
This must match Control Panel to prevent upgrade loops.